packages/paste-core/primitives/combobox/stories/index.stories.tsx

```diff
@@ -203,14 +203,27 @@ export const BasicMultiCombobox: React.FC = () => {
     highlightedIndex,
     isOpen,
     selectedItem,
+    selectItem,
   } = useComboboxPrimitive({
     items: filteredItems,
-    onSelectedItemChange: (args) => {
-      handleSelectItemOnClick(args);
+    initialInputValue: '',
+    onSelectedItemChange: ({selectedItem: selected}) => {
+      if (selected != null) {
+        handleSelectItemOnClick(selected);
+      }
+
+      selectItem((null as unknown) as string);
     },
   });
   const uid = useUID();
 
+  const dropdownProps = getDropdownProps({
+    preventKeyAction: isOpen,
+    ...getToggleButtonProps({tabIndex: 0}),
+  });
+
+  const toggleButtonProps = getToggleButtonProps({tabIndex: 0});
```

Comment:
Might be a nothing burger, but you call `getToggleButtonProps({tabIndex: 0})` here and in line 222. 

I only call it out as it's in the public example that we use in the docsite too.

Reply:
Yeah, that's not intentional and shouldn't be there. Will update, thanks.